regform

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+ <head>
+   <title>!DOCTYPE</title>
+   <meta charset="utf-8">
+   <script>
+		function correct(form){
+			
+			var email = form.email.value;
+			var regex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+			var pass = form.password.value;
+			var replypass = form.replypassword.value;
+			var passex = /^ ?=.*\d $/;
+			if(email == "" || email == " " || !regex.test(email) ){
+				alert("wrong e-email adress");
+			}
+			else if(pass == "" || pass == " "){
+			alert("password field is empty");
+			}
+			else if(!pass.match(/[0-9]/g)){
+			alert("password should contain at least one digit");
+			}
+			else if(!pass.match(/[A-Z]/g)){
+			alert("password should contain at least one Uppercase letter");
+			}
+			else if(!replypass.match(pass)){
+			alert("passwords is not math");
+			};
+				
+					
+		};
+		
+ 
+
+   </script>
+
+ </head>
+ <body>
+	<form action="" method="post" name="test" onsubmit="correct(this)">
```

Comment:
Do not use html attributes to add handlers, select element in script and add event listener to it